pkg/search/proxy/framework/plugins/cache/cache.go

```diff
@@ -17,49 +17,76 @@ import (
 	"k8s.io/apiserver/pkg/registry/rest"
 	"k8s.io/client-go/kubernetes/scheme"
 
+	"github.com/karmada-io/karmada/pkg/search/proxy/framework"
 	"github.com/karmada-io/karmada/pkg/search/proxy/store"
 )
 
-// cacheProxy caches resource from member clusters, and handle the read request(get/list/watch) for resources.
-type cacheProxy struct {
-	store             store.RESTReader
-	restMapper        meta.RESTMapper
-	minRequestTimeout time.Duration
+const (
+	order = 1000
+)
+
+// Cache caches resource from member clusters, and handle the read request(get/list/watch) for resources.
+// For reading requests, we redirect them to cache.
+// Users call these requests to read resources in member clusters, such as pods and nodes.
+type Cache struct {
```

Comment:
`cache` need not exported

Reply:
It's needed for Unit Test

Reply:
`https://github.com/kubernetes/kubernetes/tree/cdc25fdca2133ed59811c8dd669f66253bf67789/pkg/scheduler/framework/plugins`

official kubernetes scheduler plugins are exported.

Reply:
ok